src/components/OptionComponent/index.js

```diff
@@ -139,9 +139,9 @@ const OptionComponent = props => {
       return
     }
     if (mode == 'Buy') {
-      trade(chainId, library, poolAddress, IPool, account, symbol, selectedTokenAmount, symbolMarkPrice)
+      trade(chainId, library, poolAddress, IPool, account, symbol, selectedTokenAmount, symbolMarkPrice?.mul(bigNumberify(10050)).div(bigNumberify(10000)))
     } else {
-      trade(chainId, library, poolAddress, IPool, account, symbol, selectedTokenAmount.mul(bigNumberify(-1)), symbolMarkPrice)
+      trade(chainId, library, poolAddress, IPool, account, symbol, selectedTokenAmount.mul(bigNumberify(-1)), symbolMarkPrice?.mul(bigNumberify(9950)).div(bigNumberify(10000)))
     }
```

Comment:
trade still might fail due to the incorrect pricing calculation in the smart contract, especially for option symbols. @ykchong45 will check.

Reply:
👍  It turns out that having 0.5% slippage when trading (either long or short) will give us successful trade() call.